opentelemetry-sdk/tests/context/propagation/test_b3_format.py

```diff
@@ -100,9 +148,12 @@ def test_enabled_sampling(self):
                 FORMAT.SPAN_ID_KEY: self.serialized_span_id,
                 FORMAT.SAMPLED_KEY: variant,
             }
-            span_context = FORMAT.extract(get_as_list, carrier)
+            parent_span = trace.Span(
+                "parent", FORMAT.extract(get_as_list, carrier)
+            )
+            child_span = trace.Tracer().start_span("child", parent=parent_span)
```

Comment:
there's a lot of boilerplate in these tests now. Could they be refactored into a few test helper methods?

Reply:
Yes, refactored :+1: